docs/examples/spatial_transformation/histogram_matching.jl

```diff
@@ -0,0 +1,67 @@
+# ---
+# cover: assets/chelsea.gif
+# title: Histogram Matching
+# author: Ashwani Rathee
+# date: 2021-03-23
+# ---
+
+# This demo demonstrates the feature of histogram matching.
+
+# Purpose of Histogram matching is to transform the intensities in a source image so that the
+# intensities distribute according to the histogram of a specified target image. 
+
+# Here, histogram matching is shown for the RGB image. But, it can also be applied on 
+# grayscale images too. Histogram matching can be used to normalize two images when the images 
+# were acquired at the same local illumination (such as shadows) over the same location,
+# but by different sensors, atmospheric conditions, or global illumination. 
+
+using ImageContrastAdjustment
+using Images
+using Plots
+using TestImages
+
+# Load example images: one source image and one reference image 
+
+# !!! note
+#     You need to use `julia >= v"1.3.0"` and `TestImages >= v"1.3.1"` in order to load these
+#     two test images.
```

Comment:
Instead of saying a "recent TestImages", saying the version is more accurate a way.